app/domain/authentication/authn_jwt/jwt_authenticator_input.rb

```diff
@@ -0,0 +1,18 @@
+module Authentication
+  module AuthnJwt
+    # Data class to store data regarding jwt token that is needed during the jwt authentication process
+    # :reek:TooManyInstanceVariables and :reek:TooManyParameters
+    class JWTAuthenticatorInput
```

Comment:
do we have a plan to unify between this class and `AuthenticatorInput`? if so, can we please create a JIRA issue for RnD Boost and link to it here?

Reply:
But this class is good for jwt authenticators. Other authenticatiors don't need this . So i think they can remain seperatly